food-truck-api/src/main/java/food/truck/api/search/TruckSearchService.java

```diff
@@ -0,0 +1,48 @@
+package food.truck.api.search;
+
+import food.truck.api.truck.Truck;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j;
+import org.apache.lucene.search.Query;
+
+import org.hibernate.search.jpa.FullTextEntityManager;
+import org.hibernate.search.jpa.FullTextQuery;
+import org.hibernate.search.jpa.Search;
+import org.hibernate.search.query.dsl.QueryBuilder;
+import org.springframework.stereotype.Component;
+
+
+
+import javax.persistence.EntityManager;
+
+import java.util.List;
+
+@Component
+@Log4j
+@RequiredArgsConstructor
+public class TruckSearchService {
+
+    private final EntityManager entityManager;
```

Comment:
Same question as above: where's this getting instantiated? Since you posted it, I'm assuming it works, I'm kind of just curious at this point.